data/json/monsters/zed_explosive.json

```diff
@@ -132,6 +132,12 @@
     "grab_strength": 30,
     "special_attacks": [ { "id": "grab" }, { "id": "scratch_humanoid" }, { "id": "bite_humanoid" } ],
     "death_drops": "default_zombie_items",
+  "emit_fields": [
+    {
+      "emit_id": "emit_hot_air2_stream",
+      "delay": "10 s"
+    }
+  ],
```

Comment:
I don't think the hot air idea is bad, but there are probably still ways it could be bypassed, like simply dragging a tub outside and surrounding it with military searchlights. If gasoline zombies have to remain a thing, the only way to absolutely prevent people milking them for gasoline is for them to either not make puddles at all or to make what they drop unusable.

In my testing, the 250ml puddles they produce aren't even enough to actually make fires spread along them, if you ignite them with a lighter the flames will just go out by themselves even when there are multiple adjacent tiles with 250ml of gasoline on them.

Reply:
Why not just make it so gasoline zombies have a finite amount of gasoline? Like how players used to get a lot of ammunition from a single talon, and so they'd be set for a long time, and that was nerfed so talons are still a considerable threat, but no longer a loot pinata? I'm saying this as a current gasoline zombie farmer aha...that'd stop me in my tracks. It takes a considerable amount of setup and time to set up the farm. It's just not worth it after a certain point.

Reply:
Having the volume that can be dripped eliminated would also be perfectly fine. But this change is also in line with the reality that they shouldn't be dripping pure unadulterated gasoline in the first place.